frostsnapp/native/src/wallet.rs

```diff
@@ -519,8 +557,8 @@ impl _Wallet {
                     .api(&tx)
                     .push(&bincode::serde::Compat(changes.graph))?;
             }
-            tx.take_index(self.keychain_handle)
-                .extend(changes.indexer.0)?;
+            // it's unlikely to have any changes but just in case
+            self.consume_spk_revelation_change(tx, changes.indexer)?;
```

Comment:
Could you help me understand this one?

Reply:
ill try and rewrite this so the comment goes away by having a method that applies a changeset. It's unlikely that there are any changes related to spk revelation in the preceding operations.